### ISSUE TITLE
Folder overview by name sorts folders first with depth = 0

### ISSUE THREAD

**Issue by benbarbour** [LostPaul/obsidian-folder-notes#127]
I would expect (or like an option) to have everything sorted alphabetically.

- A.md
- B/
-- B.md

Displays as:

-B
-A

in the Folder Overview if the depth is set to 0.

**Comment by Mauwzola** [comment]
I agree. The "problem" also exists at depth 1. I realize that Obsidian sorts by type and then name, but it would be great if Folder Notes had the option to sort solely on name.

**Comment by LostPaul** [comment]
Will be fixed with the next update
